.github/workflows/release.yml

```diff
@@ -160,4 +160,10 @@ jobs:
           files: |
             artifacts/*.*
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: get_version
+        run: |
+          echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Send PR
+        run: |
+          curl --request POST --url 'https://api.github.com/repos/StirlingLabs/XKCP.NET/dispatches' --header 'authorization: token ${{ secrets.TK_TOKEN }}' --data '{"event_type": "release", "client_payload": {"tag":"${{ env.RELEASE_VERSION }}"}' --header "Accept: application/vnd.github.everest-preview+json"
```

Comment:
This triggers a workflow instead of pushing a PR from here, ok...

Is there a way to create the PR remotely, like say https://cli.github.com/manual/gh_pr_create or something?
> GitHub CLI comes pre-installed in all GitHub-Hosted Runners.

Reply:
I couldn't find an action that does this ready.  I preferred a workflow that can be run with a webhook. Because we can trigger (maybe) other Repositories from the source repo. Here pr settings may need to be changed. 
I can also change it with "GitHub Cli" if you want.

Reply:
Triggering the PR by remote workflow is fine, this works around any theoretical permission issues there might be. Let's get the XKCP.NET repo side up first.

Reply:
I named the token DAEMON_HOOKS so we need to refer to secrets.DAEMON_HOOKS

Reply:
```suggestion
      - name: Send PR
        run: |
          curl --request POST --url 'https://api.github.com/repos/StirlingLabs/XKCP.NET/dispatches' --header 'authorization: token ${{ secrets.DAEMON_HOOKS }}' --data '{"event_type": "release", "client_payload": {"tag":"${{ env.RELEASE_VERSION }}"}' --header "Accept: application/vnd.github.everest-preview+json"
```